master_middleman/source/subnavs/apache-hawq-nav.erb

```diff
@@ -263,6 +263,7 @@
                   <li><a href="/20/datamgmt/load/g-url-based-web-external-tables.html">URL-based Web External Tables</a></li>
                 </ul>
               </li>
+              <li><a href="/20/datamgmt/load/g-register_files">Registering Files into HAWQ Internal Tables</a></li>
```

Comment:
Jane - this entry doesn't look like a valid href (no .html) and it seems like a duplicate of the later entry.